development/ts-migration-dashboard/scripts/build-app.ts

```diff
@@ -5,8 +5,10 @@ import { hideBin } from 'yargs/helpers';
 import chokidar from 'chokidar';
 import browserify from 'browserify';
 import pify from 'pify';
-import endOfStream from 'end-of-stream';
-import pump from 'pump';
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore-error types/readable-stream.d.ts does not get picked up by ts-node
```

Comment:
You can use `ts-node --files` for this.

Reply:
@Mrtenz Oh. TIL. Is there any good reason why we are not already using `--files` for every existing call to `ts-node`or can I just change all of them? Seems like it should be the default.

Reply:
- #24807